testsuite/docs/cucumber-steps.md

```diff
@@ -613,6 +625,44 @@ For example:
   When I call actionchain.add_package_install()
 ```
 
+<a name="b13" />
+
+#### Virtualization
+
+* Create a test virtual machine on a given host
+
+The virtual machine is created without SUSE Manager, directly on the virtual host
+using `qemu-img` and `virt-install`
+
+```cucumber
+  When I create "test-vm" virtual machine on "virt-server"
+```
+
+* Assert a given virtual machine to be in a given state
+
+The possible states are `shut off`, `running` and `paused`.
+
+```cucumber
+  Then I should see "test-vm" virtual machine "shut off" on "virt-server"
+```
```

Comment:
Remove quotes around shut off ?

Reply:
`shut off` is a value taken from virsh output, we could remove the quotes and only accept `paused`, `running` and `shut off` values there